src/pages/home/Home.page.tsx

```diff
@@ -49,6 +49,13 @@ const statData: StatData[] = [
     icon: HiOutlineMail,
     percentage: "30%",
   },
+  {
+    id: 3,
+    label: "Total de orçamentos",
+    score: 100,
+    icon: HiOutlineMail,
+    percentage: "30%",
+  },
 ];
 
 const Card = ({ data }: { data: StatData }) => (
```

Comment:
Tip - Ensure proper PropType or TypeScript validations for the `data` prop in `Card` component to prevent potential issues with data type mismatches. This practices can help catch errors early and improve code reliability.